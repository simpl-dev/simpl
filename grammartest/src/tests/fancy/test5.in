foo4 foof a b c d
