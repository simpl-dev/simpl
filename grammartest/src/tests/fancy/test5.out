0:16:Foo4 {
  a=5:8:Id {
    text="foof"
  }
  x=List {
    10:10:Id {
      text="a"
    }
    14:14:Id {
      text="c"
    }
  }
  y=List {
    12:12:Id {
      text="b"
    }
    16:16:Id {
      text="d"
    }
  }
}
